.cz-config.js

```diff
@@ -1,21 +1,25 @@
-const { execSync } = require('child_process');
+const fs = require('fs')
+const { globSync } = require('glob')
+
 
 const getPackages = () => {
-  let outputPackages;
+    const ignore = [
+        'apps/examples/**',
```

Comment:
Why do we need to ignore `apps/examples/**`?

Reply:
I'm not sure, but I believe it's being filtered for convenience since we don't update them as often... Likely @siggerzz  can help clarify that.

Reply:
Yeh, that's right – we don't add changesets for the example apps, we add a changeset to the root monorepo for these changes (mainly as they start to clog up the commitizen UI otherwise with lots of apps that we only use for testing atm).